app/views/users/edit.html.haml

```diff
@@ -0,0 +1,8 @@
+- content_for(:page_title) do
+  %h1 Edit Profile
+
+= simple_form_for @user, url: '/user', method: :put  do |f|
```

Comment:
URL helper must be passed in for singular resource routes, as per:

https://github.com/rails/rails/issues/1769

and:

http://guides.rubyonrails.org/routing.html#singular-resources